model/browser/events.yaml

```diff
@@ -0,0 +1,82 @@
+groups:
+  - id: event.browser.user_action
+    stability: development
+    type: event
+    name: browser.user_action
+    brief: >
+      This event describes actions performed by the user such as click, scroll, zoom, resize, etc.
+    body:
+      id: browser.user_action
+      requirement_level: required
+      stability: development
+      type: map
+      fields:
+        - id: element
+          type: string
+          stability: development
+          brief: Target element tag name and it is obtained via event.target.tagName.
+          requirement_level: recommended
+          examples: ["button"]
+        - id: element_xpath
+          type: string
+          stability: development
+          brief: Target element xpath
+          requirement_level: recommended
+          examples: ["//*[@id='testBtn']"]
+        - id: user_action_type
+          type: enum
```

Comment:
need to confirm the `user_action_type` enum. Currently, I am using the definitions from https://github.com/microsoft/ApplicationInsights-JS/blob/main/extensions/applicationinsights-clickanalytics-js/src/Enums.ts

Reply:
will change the enum to long name